tests/helpers.py

```diff
@@ -24,12 +25,18 @@ async def create_postgres_queue(**kwargs: t.Any) -> PostgresQueue:
         ),
     )
     await queue.connect()
+    await queue.upkeep()
+    await asyncio.sleep(0.1)  # Give some time for the tasks to start
```

Comment:
Without this, some tests were flaky because tearDown would be called before some upkeep tasks could start.